build/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>cloud-openstack</artifactId>
         <groupId>jetbrains.buildServer.clouds</groupId>
-        <version>1.3-SNAPSHOT</version>
+       <version>1.3-SNAPSHOT</version>
```

Comment:
If some code change (SonarQube quality flaw, ...), a space ` <version>1.3-SNAPSHOT</version>` could perhaps be added (x5 files), to not introduce some changes if there are not, better for commit visibility :wink:.